sirepo/package_data/static/js/sirepo.js

```diff
@@ -1279,20 +1279,23 @@ SIREPO.app.factory('frameCache', function(appState, panelState, requestSender, $
         }
         var isHidden = panelState.isHidden(modelName);
         var frameRequestTime = new Date().getTime();
+        var waitedTimeElapsed = false;
```

Comment:
Use let. In general almost all "vars" can be replaced by "let" or "const". I tend to replace them as I go (not in the entire file) but YMMV.

Also, it's not major but I prefer names for booleans that are yes/no questions ("hasWaitTimeElapsed"). I don't always follow that myself though :|